app/models/article.rb

```diff
@@ -1,4 +1,10 @@
 class Article < ApplicationRecord
+  def to_meta_tags
```

Comment:
move this method down the file. does not need to be so high up. could be the last thing in the file.